lib/blacklight/maps/maps_search_builder.rb

```diff
@@ -0,0 +1,20 @@
+module BlacklightMaps
+  module MapsSearchBuilder
```

Comment:
I'm all for aligning the patterns as closely as possible between GeoBL and blacklight-maps. So the logic then gets included like so: https://github.com/geoblacklight/geoblacklight/blob/master/lib/geoblacklight/search_builder.rb#L3?